metabase/metabase
# Update default and min size constraints for dashboard cards

Closes https://github.com/metabase/metabase/issues/31408

### Description

Updates the minSize and defaultSize for the visualizations based on the [product doc](https://www.notion.so/metabase/Update-default-and-min-size-constraints-for-dashboard-cards-46d75db54e864541b75f33f1e6301628?pvs=4).

### How to verify

Feel free to go into the product doc and add different types of questions - when you add a question, they should conform to the new size standard.

If users have cards that are smaller than the new minSize, they will be allowed to keep a size smaller than the minSize until they save the dashboard. After the dashboard is saved, then they will not be allowed to resize the card below the new constraints.


### Checklist
